api/signer-api.yml

```diff
@@ -593,6 +633,185 @@ paths:
               schema:
                 type: string
                 example: "OK"
+  /reload:
```

Comment:
We discussed some time ago if we should include non Signer API endpoints here or not. We can discuss it again though, if you have another point of view

Ref: https://github.com/Commit-Boost/commit-boost-client/pull/231#discussion_r1924129461